### ISSUE TITLE
Can't get any command to work

### ISSUE THREAD

**Issue by osteel** [NodeRedis/node_redis#106]
Hi,

I have some difficulties to get any command to work. Warning: I'm a linux n00b, and installing stuff is a pain in the ass for me.

I have node running as well as connect and socket.io. I a priori correctly installed node_redis and connect-redis (https://github.com/visionmedia/connect-redis) with NPM. Here are my required modules in my server.js file:

var
querystring      = require('querystring'),
    url          = require('url'),
    connect      = require('/root/node-v0.4.7/node_modules/connect/');
    redis                = require('/root/node-v0.4.7/node_modules/redis/'),
    rc                   = redis.createClient(),
    RedisStore    = require('/root/node-v0.4.7/node_modules/connect-redis/'),
    sessionStore= new RedisStore();

First of all, I don't understand how to install stuff so that I could just do require('redis') instead of specifying the complete path, for example. But, as it's working this way, I didn't pay attention to this any longer.

Later in my code, I use connect-redis to store sessions:

var server = connect.createServer(
    connect.cookieParser(), 
    connect.session({secret: 'whocares', store: sessionStore}),
...

The thing is, this works great and I can then retrieve sessions with sessionStore.get(sessionId, function(error, data) {...});

But, when I try to use rc, nothing seems to work:

rc.set("test", "value", redis.print);
console.log(rc.get("test"));

All I get is "undefined" (whatever is the command I try to use).

I assume it is necessary to install redis first (again, I may write dumb things, but linux world is really foggy to me), so I did it following this: http://redis.io/download and, as I was so far able to deal with connect-redis and sessions, I assumed I had done it the right way.

So, I have node and its modules installed in /root/node-v0.4.7/ and redis installed in ~/redis-2.2.8/. 

To sum up, I'm not pretty sure I did install the whole thing the correct way and, if ever I did, do you see anything I might do wrong? Do I have to "match" redis and redis_node in any way?

Thanks,

OSteEL
